joomla/joomla-cms
# Remove unused enableSites() method

Pull Request for Issue #26189 

### Summary of Changes
Removes the unused method `enableSites()`

### Testing Instructions
1. Apply PR
2. Go to System -> Update Sites
3. Check that Enable and Disable of Update sites still works

### Expected result
Sites are disabled and enabled as expected.

### Documentation Changes Required
None

@brianteeman
